process/dns_v1.go

```diff
@@ -0,0 +1,336 @@
+package process
+
+import (
+	"bytes"
+	"encoding/binary"
+	"math"
+
+	"github.com/DataDog/mmh3"
+)
+
+// DNS data is encoded as a very basic bucketed hash table.  There are three blocks, or buffers, of data:
+//
+//	The "name" block is all of the unique DNS names.  The length of the name is stored as a varint
+//	followed by the name itself
+//
+//	The "bucket" block contains all of the hash buckets.  The format of each bucket is:
+//		varint for number of entries in bucket
+//		For each entry in the bucket:
+//			varint for length of key
+//			key bytes
```

Comment:
nit: key represents an IP address, correct? If so, why not using a less cryptic name? 😛

Reply:
Can do.  I was using generic hash table nomenclature here but there really is no point.